### ISSUE TITLE
RNFIRMessaging.h not found

### ISSUE THREAD

**Issue by jreziga** [evollu/react-native-fcm#156]
Hello,

I've an issue when inserting RNFIRMessaging.h in my AppDelegate.m (using react 0.35 and xcode 8)
1. I added Firebase/Messaging, uncommented use_framework
2. I installed library with npm and linked with rnpm (RNFIRMessaging.Xcodeproj is in Librarires and RNFIRMessaging.a is linked in my project's target)
3. When I insert #import "RNFIRMessaging.h" I got this issue: 

![Image of error]
(https://puu.sh/rHAlh/1f267f533f.png)

Any idea ?

Thanks !

**Comment by tioback** [comment]
I couldn't find a fix for this, so I gave up and reinstalled it manually.

**Comment by the-simian** [comment]
Hey I think this worked: https://github.com/evollu/react-native-fcm/issues/21

**Comment by evollu** [comment]
@the-simian if this fix works, it means the link function has a bug

**Comment by the-simian** [comment]
Hey the link function does have a problem! I've been able to zero in on it, but I have to admit I do n't fully understand what the fix is. I'll link to the thread that had it. 

https://github.com/facebook/react-native/issues/10032  <---- This. Seems related to an older/version and newer version of... this (https://github.com/CocoaPods/CocoaPods/issues/1275) 

I actually got past this specific error in this thread, but now my project doesn't see FirebaseAnalytics (or any of the Firebase modules, tbh). And i don't beleive I have a malformed podfile, and I've dome most of the 'fixes' I've seen on SO too. I am not sure if this could also be a side-effect of bad linking, or somethign else. If I get it working I'll post what worked.
